testsuite/ui/views/admin/audience/developer_portal/__init__.py

```diff
@@ -72,7 +72,7 @@ def is_displayed(self):
 class ActiveDocsNewView(BaseAudienceView):
     """View representation of New Active Docs page"""
     path_pattern = '/admin/api_docs/services/new'
-    create_spec_btn = ThreescaleCreateButton()
+    create_spec_btn = GenericLocatorWidget("//button[contains(text(), 'Create')]")
```

Comment:
Why this change?

Reply:
Oas3 json contains word Create so we need to specify create button locator that it contains //button instead of //*